packages/blade/src/components/Tooltip/_decisions/decisions.md

```diff
@@ -0,0 +1,199 @@
+
+<img src="./tooltip-thumbnail.png" width="380" alt="" />
+
+- [Design](#design)
+- [`Tooltip` API](#tooltip-api)
+  - [Examples:](#examples)
+    - [Usage](#usage)
+  - [Working with non-interactive triggers](#working-with-non-interactive-triggers)
+  - [Custom Triggers](#custom-triggers)
+- [Library](#library)
+  - [Bundle Size](#bundle-size)
+- [Implementation detail nuances:](#implementation-detail-nuances)
+- [Motion](#motion)
+- [Accessibility](#accessibility)
+- [Open Questions](#open-questions)
+- [References](#references)
+
+## Design
+
+[Figma Link](https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade---Payment-Light?type=design&node-id=40540-559304&t=tmTrf3xJU6oj59fM-0) to all variants of the Tooltip component
+
+## `Tooltip` API
+
+| Prop               | Type                                                                     | Default     | Description                                                                                                                                              | Required |
+| ------------------ | ------------------------------------------------------------------------ | ----------- | -------------------------------------------------------------------------------------------------------------------------------------------------------- | -------- |
+| content            | `string`                                                                 | `undefined` | Content of the tooltip                                                                                                                                   | ✅       |
+| children           | `React.ReactNode`                                                        | `undefined` | Trigger component for tooltip, Accepts any interactive element or icons                                                                                  | ✅       |
+| placement          | `top, top-start, top-end, left, right, bottom, bottom-start, bottom-end` | `top`       | Placement of tooltip                                                                                                                                     |          |
+| shouldWrapChildren | `boolean`                                                                | `false`     | If set to true, the tooltip trigger will be wrapped in a interactive `<div>`, this is useful for making non-interactive elements work as tooltip trigger |          |
+| onOpen             | `Callback`                                                               | `undefined` | Called when tooltip is opened                                                                                                                            |          |
+| onClose            | `Callback`                                                               | `undefined` | Called when tooltip is closed                                                                                                                            |          |
+
```

Comment:
what happens when you render a disabled element say button? will the tooltip be visible?
```
<Tooltip content="Amount reversed to customer bank account" placement="top">
  <IconButton disabled icon={InfoIcon} />
</Tooltip>
```

Reply:
Yes disabled elements can also have hover information associated with them, imagine a button with a tooltip showing some additional context on why the button is disabled.

Reply:
What if we only want to show a tooltip only when the button is disabled 
We can pass a boolean isDisabled to the button - but we will always have to wrap the button in a tooltip ...
will we have to pass either content as blank or non-blank to achieve this ?